lib/longtail/stackoverflow/posts.pl

```diff
@@ -61,7 +61,31 @@
     }
 
     print qq(\nTYPE: $name\t$data->{src_domain}\n);
-    
+
+    my %post_links = ();
+    if (-e "$stack_dir/$data->{src_domain}/PostLinks.xml") {
```

Comment:
I think we want to process the files even if PostLinks.xml isn't found. I think the post links can be classified as non-essential information to the IA itself.